packages/react-components/src/components/styles/LoadingSpinner.style.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { mergeStyles } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const loadingSpinnerContainerStyle = (): string =>
```

Comment:
This positions the spinner centrally on top of anything its contained in, does this mean if there's a freeze frame of a video we will see the spinner on top of the freeze frame?
(My mental image of this would be to replace the video tile contents with a spinner)

Reply:
Yep, iv added some screen recordings in the PR description so you can see how it works

Reply:
ill prob refactor this a bit and get rid of LoadingSpinner.tsx, i dont really think i need it